app/components/works/show/content_file_download_button_component.rb

```diff
@@ -19,11 +19,6 @@ def call
                                                      link: download_content_file_path(content_file),
                                                      data:, **opts)
       end
-
-      def render?
-        File.exist?(StagingSupport.staging_filepath(druid: content_file.content.work.druid,
-                                                    filepath: content_file.filepath))
-      end
```

Comment:
we will always show download links since if we list a file, it should be on staging or in preservation